README.md

```diff
@@ -125,8 +131,11 @@ Once in the web app:
 
 ***Answer***: Chunking allows us to limit the amount of information we send to OpenAI due to token limits. By breaking up the content, it allows us to easily find potential chunks of text that we can inject into OpenAI. The method of chunking we use leverages a sliding window of text such that sentences that end one chunk will start the next. This allows us to reduce the chance of losing the context of the text.
 
-### Troubleshooting
+***Question***: How can we upload additional PDFs without redeploying everything?
```

Comment:
This is where we could use summary/details elements instead, to hide the answers by default.